vm-test-build-copy.sh

```diff
@@ -56,8 +56,8 @@ build_and_copy_composability() {
    contract_with_underscores="${contract//-/_}"
 
    sc-meta all build --target-dir $TARGET_DIR --path ./contracts/feature-tests/composability/$contract || return 1
-   cp -R contracts/feature-tests/composability/$contract/output/${contract}.wasm \
```

Comment:
Add a
`rm contracts/feature-tests/composability/$contract/output/*.wasm`
would be nice to automatically get rid of the old files.